src/LiveDevelopment/Documents/HTMLDocument.js

```diff
@@ -63,7 +63,7 @@ define(function HTMLDocumentModule(require, exports, module) {
 
         this.editor = editor;
         this._instrumentationEnabled = false;
-
+        this._debug = false;
```

Comment:
I'm not familiar with the `XXXBramble` prefix, is this a brackets convention or us?